src/web/auth.rs

```diff
@@ -176,7 +176,7 @@ impl<'r> FromRequest<'r> for Admin {
                 let mut sm = sm_lock.lock().await;
                 let un = state
                     .get_user(uid)
-                    .map(|u| u.name)
+                    .map(|u| u.global_name.unwrap_or(u.name))
```

Comment:
this and the next change are drive bys to do with the discord username change (`user.name` is no longer the best option to choose for a default display name)